src/graph.rs

```diff
@@ -2474,6 +2474,27 @@ pub(crate) fn parse_js_module_from_module_info(
             maybe_npm_resolver,
           );
           dep.maybe_deno_types_specifier = Some(specifier_text);
+        } else {
+          let types_resolution = resolve(
+            &specifier_text,
+            range.clone(),
+            ResolutionMode::Types,
+            jsr_url_provider,
+            maybe_resolver,
+            maybe_npm_resolver,
+          );
+          if types_resolution.maybe_specifier()
+            != dep.maybe_code.maybe_specifier()
+          {
+            dep.maybe_type = resolve(
+              &specifier_text,
+              range.clone(),
+              ResolutionMode::Types,
+              jsr_url_provider,
+              maybe_resolver,
+              maybe_npm_resolver,
+            );
```

Comment:
Is this just `dep.maybe_type = types_resolution`?

Reply:
Yes, thanks!